Source/CarthageKit/GitHub.swift

```diff
@@ -38,6 +38,11 @@ public struct GitHubRepository: Equatable {
 		return GitURL("ssh://git@github.com/\(owner)/\(name).git")
 	}
 
+	/// The URL for filing a new GitHub issue for this repository.
+	public var newIssueURL: NSURL? {
```

Comment:
I think we could make this non-optional, since the URL should be valid if this really refers to a GitHub repository (if it doesn't, that's a problem for initialization time).